owncloudData/src/main/java/com/owncloud/android/data/files/repository/OCFileRepository.kt

```diff
@@ -460,11 +460,11 @@ class OCFileRepository(
         if (!anyConflictInThisFolder) {
             remoteFolder.etagInConflict = null
         }
-
-        return localFileDataSource.saveFilesInFolderAndReturnThem(
+        localFileDataSource.saveFilesInFolderAndReturnThem(
```

Comment:
Just 1 thing before merging this: the `saveFilesInFolderAndReturnThem` result is never used after changing this, we could transform it to `saveFilesInFolder`. Adapt the existing tests as well

Reply:
I have realized that we need the return of that function. Because if not in the current code if we upload a file to a folder available offline, the file id will be null and it will not download the file. So I am going to change this function `insertFilesInFolderAndReturnThem` to return only the changed files.